src/views/CashlinkReceive.vue

```diff
@@ -0,0 +1,682 @@
+    .skip:focus {
+        outline: none;
+        opacity: 1;
+        text-decoration: none;
+    }
+
+    .skip:hover .nq-icon,
+    .skip:focus .nq-icon {
+        transform: translate3D(-0.25rem, 0, 0);
+    }
+
+    .status-screen {
+        position: absolute;
+        transition: opacity .4s;
+    }
+
+    .overlay {
+        position: absolute;
+        top: 0;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        transition: opacity .4s;
+        background: rgba(255, 255, 255, .875); /* equivalent to keyguard: .5 on blurred and .75 on account details */
+    }
+
+    .overlay .close {
+        position: absolute;
+        right: 2rem;
+        top: 2rem;
+        font-size: 3rem;
+        padding: 0;
+        height: 3rem;
+        background: none;
+    }
+
+    .overlay .close .nq-icon {
+        opacity: 0.2;
+        transition: opacity .25s;
+    }
+
+    .overlay .close:hover .nq-icon,
+    .overlay .close:focus .nq-icon {
+        opacity: 0.4;
+    }
+
+    .account-selector {
+        margin-top: -3rem;
+    }
+
+    .account-selector >>> .amount {
+        display: none !important;
+    }
+
+    .blur-target {
+        transition: filter .4s;
+    }
+
+    .account-selector-shown .blur-target {
+        filter: blur(20px);
+    }
+    .account-selector-shown .page-footer {
+        filter: blur(35px);
+    }
+
+    @media (max-width: 939px) {
+        .container {
+            padding: 0 4rem;
+        }
+
+        .welcome-text {
+            margin-left: 5.75rem;
+        }
+
+        .welcome-text .nq-h1 {
+            font-size: 5.5rem;
+            margin-bottom: 3rem;
+        }
+
+        .welcome-text .nq-text {
+            font-size: 3rem;
+        }
+    }
+
+    @media (max-width: 799px) {
+        .container {
+            padding: 0 2.5rem;
+        }
+
+        .welcome-text {
+            margin-left: 3.75rem;
+        }
+
+        .welcome-text .nq-h1 {
+            font-size: 4rem;
+            margin-bottom: 1.75rem;
+        }
+
+        .welcome-text .nq-text {
+            margin-top: 1.75rem;
+            font-size: 2.5rem;
+        }
+    }
+
+    @media (max-width: 699px) {
+        .container {
+            flex-direction: column-reverse !important;
+            padding: 0;
+        }
+
+        .small-page {
+            height: auto !important;
+            flex-grow: 1;
+        }
+
+        .top-spacer {
+            display: none;
+        }
+
+        .no-account .value {
+            font-size: 6rem;
+        }
+
+        .no-account .data {
+            font-size: 2rem;
+        }
+
+        .welcome-text {
+            max-width: 420px;
+            text-align: center;
+            margin: 3rem 0 4rem;
+            padding: 0 2rem;
+        }
+
+        .transition-welcome-text-enter,
+        .transition-welcome-text-leave-to {
+            transform: translate3d(0, 3rem, 0);
+        }
+    }
+
+    @media (max-width: 450px) {
```

Comment:
Should reduce the blur radius on mobile.

Reply:
@danimoh The `SignTransactionLedger` component has these same blur values and does not reduce them on mobile ;)

Reply:
Shame on the author of that component!